ci/build/npm-postinstall.sh

```diff
@@ -36,6 +36,12 @@ vscode_yarn() {
   yarn --production --frozen-lockfile
   cd extensions
   yarn --production --frozen-lockfile
+  for ext in */; do
+    ext="${ext%/}"
+    echo "extensions/$ext: installing dependencies"
+    yarn --production --frozen-lockfile
```

Comment:
I think there's a missing `cd` into the extension dir.

Reply:
🤦🏼